tee-worker/client-api/parachain-api/prepare-build/interfaces/vc/definitions.ts

```diff
@@ -242,7 +242,7 @@ export default {
         },
         // PlatformUserType
         PlatformUserType: {
-            _enum: ["KaratDaoUser", "MagicCraftStakingUser"],
+            _enum: ["KaratDao", "MagicCraftStaking", "DarenMarket"],
         },
```

Comment:
we will need to update client-api, but not a breaking change for client-sdk nor clients in general.